ingestion_program/ingestion_parallel.py

```diff
@@ -403,7 +404,8 @@ def predict_submission(self, test_settings):
         # create a product of set and test set indices all combinations of tuples
         all_combinations = list(product(set_indices, test_set_indices))
         # randomly shuffle all combinations of indices
-        np.random.shuffle(all_combinations)
+        random_state_initial = np.random.RandomState(initial_seed)
```

Comment:
use same name for random state `random_state` as in ingestion

Reply:
They should be same. because Its copy paste. the difference is in ingestion_parallel its in 2 different places 
which in ingestion.py its next to each other like there is 2 random state